backend/routes/download.js

```diff
@@ -34,13 +41,298 @@ const ensureAdminDownloadToken = (req, res, next) => {
+            ])
+              .then(([images, videos, otherMedia, submissionsWithSubmitters]) => {
+                // Create a mapping for type objects to their ids for easy assigning
+                const imageIdsToImage = images.reduce((obj, image) => ({
+                  ...obj,
+                  [image.id]: image
+                }), {})
+                const videoIdsToImage = videos.reduce((obj, video) => ({
+                  ...obj,
+                  [video.id]: video
+                }), {})
+                const otherMediaIdsToImage = otherMedia.reduce((obj, other) => ({
+                  ...obj,
+                  [other.id]: other
+                }), {})
+                // Format the data to be csv-stringified
+                return submissionsWithSubmitters.reduce((arr, { user, group, entries }) => {
+                  // create update entry objects that contain modified entry data plus:
+                  // path, vert and horiz dementions medaiType, videoUrl
+                  const newSubmissionSummaries = entries.map(entry => {
+                    let entryData = entry.dataValues
+                    let entryType = entry.entryType === IMAGE_ENTRY ? 'Image'
+                      : entry.entryType === VIDEO_ENTRY ? 'Video'
+                        : entry.entryType === OTHER_ENTRY ? 'OtherMedia' : ''
+                    let newEntry = {
+                      studentEmail: `${entryData.studentUsername}@rit.edu`,
+                      studentFirstName: user ? user.firstName : null,
+                      studentLastName: user ? user.lastName : null,
+                      isGroupSubmission: entryData.groupId ? true : null,
+                      groupParticipants: group ? group.participants : null,
+                      entryType: entryType,
+                      title: entryData.title,
+                      comment: entryData.comment,
+                      moreCopies: entryData.moreCopies,
+                      forSale: entryData.forSale,
+                      awardWon: entryData.awardWon,
+                      invited: entryData.invited,
+                      yearLevel: entryData.yearLevel,
+                      academicProgram: entryData.academicProgram,
+                      excludeFromJudging: entryData.excludeFromJudging,
+                      submittedAt: moment(entryData.createdAt).format(),
+                      path: '',
+                      horizDimInch: '',
+                      vertDimInch: '',
+                      mediaType: '',
+                      videoUrl: ''
+                    }
+                    // Add entry data to data object
+                    if (entry.entryType === IMAGE_ENTRY) {
+                      let imageObj = imageIdsToImage[entry.entryId]
+                      // TODO: change loclahost for deployment
+                      return {
+                        ...newEntry,
+                        path: `//localhost:3000//static/uploads/${imageObj.path}`,
```

Comment:
Hate to nit here, but let's just start using a config option for this now, since we'd have to do it in a few days anyway. I think just change `backend/.env-sample` to include an all-caps key like uhhhh `STATIC_UPLOAD_BASE_URL`. Be sure to update your `backend/.env` too. And then boom, here we can use
```javascript
`${ config.get('STATIC_UPLOAD_BASE_URL') }/${ imageObj.path }`
```
right here, I think

Also, the `//localhost/...` trick only works in browsers I think (?), the config should probably be `http://localhost/...`